src/frontend/themes.scss

```diff
@@ -91,6 +91,10 @@ body.midnightMirage {
   /* Others for compatibility, probably can be refactored */
   --primary: var(--brand-primary);
   --secondary: var(--brand-secondary);
+
+  /* AlphabetFilter */
+  --filter-accent-color: var(--accent);
+  --filter-accent-hover: var(--accent-overlay, var(--accent));
```

Comment:
I don't think we need this generic css variables in every theme, they are all the same, can't they be defined once in the root?